src/components/FilterGroup.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from "react";
+import { type a } from "@chakra-ui/checkbox/dist/checkbox-types-a3d7c663";
```

Comment:
this looks weird? importing a file with hash in its name doesn't look right

Reply:
oh true, that must have been an auto import/fix. should be `UseCheckboxProps`